utilities/CojUtilities.ts

```diff
@@ -1,16 +1,36 @@
 import { COJ_EPOCH, GOLD_GUIDE_VERSION_REGEX } from "./Constants";
 
+export const SIGNABLE_OFFSET = 13 * 7 * 24 * 60 * 60 * 1000; // 13 weeks.
+
 export function getStatusText(startDate: string | null) {
   if (!startDate) {
     return "Unknown status";
   }
 
-  return new Date(startDate) < COJ_EPOCH
-    ? "Submitted directly to Local Office"
-    : "Not signed";
+  if (new Date(startDate) < COJ_EPOCH) {

```

Comment:
I think getStatusText needs a bit of refactoring! We don't need all this if/else nesting given we do a return after each. Instead, just handle each condition in turn (the last condition can just be a simple return).